purrr/users.py

```diff
@@ -67,30 +116,31 @@ def github_username_to_slack_user_id(github_username: str) -> str:
             return user.id
 
     # Optimization: cache unsuccessful results too (i.e. external users).
-    data_helper.debug(f"GitHub user {gh_user_link}: email & name not found in Slack")
+    debug.log(f"GitHub user {gh_user_link}: email & name not found in Slack")
     data_helper.cache_slack_user_id(github_username, "not found")
     return ""
 
 
-def _email_to_slack_user_id(email: str) -> str:
-    """Convert an email address to a Slack user ID.
+def resolve_github_user(github_user) -> str:
```

Comment:
personally liked the old naming better. more clear as to what it's doing.

Reply:
This is just a GitHub mistake, `_email_to_slack_user_id` still exists - it just moved closer to the top. `resolve_github_user` is a new function that just happens to be in the same place.